src/http-auth-interceptor.js

```diff
@@ -5,6 +5,12 @@
  * (c) 2012 Witold Szczerba
  * License: MIT
  */
+
+/* commonjs package manager support (eg componentjs) */
+if (typeof module !== "undefined" && typeof exports !== "undefined" && module.exports === exports){
+  module.exports = 'http-auth-interceptor';
```

Comment:
Just to let me figure it out: as of now, without the `module.exports = 'http-auth-interceptor';` can you load the module by `import` statement like this?
```
import 'http-auth-interceptor';
angular.module('app', ['http-auth-interceptor']);
```
Would that work? Is the `export....` just a convenience so the `import` statement would assign a `httpAuth` value?